### PR TITLE
Add rbd disk tests for blockcommit command.

### PR DESCRIPTION
**Pull request by rbian**

### CODE REVIEW

**Comment by waynesun09** [comment]
@rbian this is duplicate with PR #563 , better close it.

**Comment by rbian** [comment]
@waynesun09 , it's not the same tests. I'd better modify the description. Thanks!

**Comment by will-Do** [comment]
@rbian 
Replace real IP address with to a sample value( 'SAMPLE.IP.COM', eg.)is necessary.

File: libvirt/tests/cfg/virsh_cmd/domain/virsh_blockcommit.cfg [hunk 1]
```diff
@@ -72,6 +72,20 @@
                                 - disk_qcow2:
                                     disk_format = "qcow2"
                                 - disk_raw:
+                        - ceph:
+                            disk_source_protocol = "rbd"
+                            disk_source_host = "EXAMPLE_HOSTS"
```
Comment (waynesun09) [inline libvirt/tests/cfg/virsh_cmd/domain/virsh_blockcommit.cfg]: better check with skip in test if default with "EXAMPLE"

**Comment by will-Do** [comment]
LGTM, I'm going to merge this PR

**Pull request merged**